share/github-backup-utils/ghe-backup-mssql

```diff
@@ -90,13 +89,69 @@ ensure_same_dbs() {
   fi
 }
 
+no_conflicting_backups_exist_db() {
+  backups_dir=$1
+  db=$2
+  ghe_verbose "Checking if the backup job has ran for '$db' since the last backup-utils backup..."
+
+  latest_full_backup=$(find "$backups_dir" -type f -name "$db*.bak" | egrep '[0-9]{8}T[0-9]{6}' | sort | tail -n 1)
+  latest_full_backup_file="${latest_full_backup##*/}"
```

Comment:
I'm currently testing each database _individually_ based on its latest backup file.

Alternatively, instead of checking each database individually I could instead find the oldest (by timestamp) full backup for _any_ database, and search everything newer. This would be faster and is probably fine since we always take backups of all DBs together... but it still seems a bit odd to me and might make future improvements like checking LSN chains harder.

@boxofyellow thoughts?

Reply:
This gets my vote.

Reply:
Per discussion we just had, will keep current behavior of individually inspecting each database. This seems like a safer approach since we want to exclude checking other databases that shouldn't be backed up (e.g. unused partition dbs or non-actions dbs). It also makes it easier to add additional checks (e.g. LSN chain) in the future.